src/main/resources/spring/camel-context.xml

```diff
@@ -208,6 +269,26 @@
             </marshal>
         </route>
 
+        <route id="administration-metrics-model-to-csv">
+            <from uri="direct:administrationMetricsModelToCsv" />
+            <marshal>
+                <bindy id="administrationMetricsModelBindyDataformat" type="Csv" classType="org.jboss.xavier.analytics.pojo.AdministrationMetricsModel"/>
+            </marshal>
+        </route>
+
+        <route id="administration-report-csv-aggregator">
+            <from uri="direct:administration-report-csv-aggregator"/>
+            <aggregate strategyRef="zipAggregationStrategy" completionSize="2">
+                <correlationExpression>
+                    <header>CSVCorrelationID</header>
+                </correlationExpression>
+                <setHeader headerName="CamelFileName">
```

Comment:
Use `Exchange.FILE_NAME` instead of its value `CamelFileName`

Reply:
done